conf.yaml

```
@@ -1321,6 +1321,11 @@ contents:
                         repo:   apm-agent-java
                         path:   CHANGELOG.asciidoc
                         exclude_branches:   [ 0.7, 0.6]
+                      -
+                        repo:   apm-aws-lambda
+                        path:   docs
+                        exclude_branches:   [ 0.7, 0.6 ]
+                        map_branches: *mapMasterToMain
```

```suggestion
                        map_branches: 
                          1.x: main
```

Or `8.1` instead of main, etc. The left side is the branch in the "branches" list of this book (and therefore corresponds to the branch apm-agent-java repo, and the right side is the branch in "this" repo (`apm-aws-lambda`)

---

When you add a 2.x branch of `apm-agent-java`, you'll just add a new key/value pair (a new line) to this mapping.

---

Neat. Thanks for the explanation. The `apm-agent-*` branches will need to be mapped to whatever `current` is. Is there a way to map to `current` so these lines don't need to be updated on a minor release?

---

Ohhhh or maybe even a new `*mapToCurrent` attribute? So that only one location needs to be updated?

---

You can probably use `stackcurrent` for this. It's defined near the top of `conf.yaml`. This would look like 

```
1.x: *stackcurrent
```

---

Nice. Thanks, Greg!